src/Spice86.Shared/Emulator/VM/Breakpoint/Serializable/ProgramSerializedBreakpoints.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Spice86.Shared.Emulator.VM.Breakpoint.Serializable;
+
+/// <summary>
+/// Represents a collection of breakpoints associated with a specific program, identified by its SHA256 hash.
+/// </summary>
+public record ProgramSerializedBreakpoints {
```

Comment:
The record name is a bit misleading. I would call it ProgramSerializableBreakpoints

Reply:
Done.